controllers/vgmanager_daemonset.go

```diff
@@ -0,0 +1,204 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	lvmv1alpha1 "github.com/red-hat-storage/lvm-operator/api/v1alpha1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	hostContainerPropagation  = corev1.MountPropagationHostToContainer
+	directoryHostPath         = corev1.HostPathDirectory
+	HostPathDirectoryOrCreate = corev1.HostPathDirectoryOrCreate
+
+	LVMdVolName   = "lvmd-conf"
+	UdevVolName   = "run-udev"
+	DevDirVolName = "device-dir"
+	SysVolName    = "sys"
+
+	LVMdDir      = "/mnt/lvmd/"
+	LVMdConfPath = "/mnt/lvmd/lvmd.conf"
+	devDirPath   = "/dev"
+	udevPath     = "/run/udev"
+	sysPath      = "/sys"
+
+	// LVMDConfVol is the corev1.Volume definition for the directory on host ("/mnt/lvmd/") for storing
+	// the lvmd.conf file
+	LVMDConfVol = corev1.Volume{
+		Name: LVMdVolName,
+		VolumeSource: corev1.VolumeSource{
+			HostPath: &corev1.HostPathVolumeSource{
+				Path: LVMdDir,
+				Type: &HostPathDirectoryOrCreate,
+			},
+		},
+	}
+
+	// LVMDConfVolMount is the corresponding mount for LVMDConfVol
+	LVMDConfVolMount = corev1.VolumeMount{
+		Name:             LVMdVolName,
+		MountPath:        LVMdDir,
+		MountPropagation: &hostContainerPropagation,
```

Comment:
- I'm a bit unsure whether host to container would suffice or not
- I saw the demo earlier and I felt it did work, can you just take a note if this propagation would be enough or we need a bi-directional
- rationale being, with current setting any mounts on host at this path will be realised in container, does the other PR is mounting anything at this path?

Reply:
I'll need to test this with the next PR.